Rubberduck.Core/AddRemoveReferences/ReferenceModel.cs

```diff
@@ -61,6 +65,30 @@ public ReferenceModel(IReference reference, int priority)
             Type = reference.Type;
         }
 
+        public ReferenceModel(ITypeLib reference)
+        {
+            var documentation = new ComDocumentation(reference, -1);
```

Comment:
Just say no to magic numbers. ;-)

Reply:
Any naming suggestions? AFAIK it isn't a const in oaidl.h, so maybe something like `TypeLibDocIndex`? I'm guessing that replacing it with `unchecked(1u << 32)` isn't what we're after here.

Reply:
I guess that name will work; we need to get in habit of using named constants, so we don't have to remember what the magical `-1` is supposed; I actually was thinking of `MEMBER_NIL` but evidently that's not the same "kind" of the magical number we want to use.